src/main/java/eu/darkbot/api/objects/Locatable.java

```diff
@@ -0,0 +1,29 @@
+package eu.darkbot.api.objects;
+
+/**
+ * Represents location point in-game.
+ */
+public interface Locatable {
+    /**
+     * Creates new instance of {@link Locatable} with given parameters.
+     * @param x coordinate
+     * @param y coordinate
+     * @return {@link Locatable} with given coordinates
+     */
+    static Locatable of(double x, double y) {
+        return new Locatable() {
+            public double getX() { return x; }
+            public double getY() { return y; }
+        };
```

Comment:
Creating an annonymous class is undesirable, if needed you can just create an inner `LocatableImpl` class that implements it